tests/test_middleware.py

```diff
@@ -57,10 +57,8 @@ def test_is_POST(self):
     @override_settings(OIDC_RP_CLIENT_ID='foo')
     @override_settings(OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS=120)
     @patch('mozilla_django_oidc.middleware.get_random_string')
-    @patch('mozilla_django_oidc.utils.get_random_string')
-    def test_is_ajax(self, mock_utils_random, mock_middleware_random):
+    def test_is_ajax(self, mock_middleware_random):
         mock_middleware_random.return_value = 'examplestring'
```

Comment:
Nit: use `side_effect` to get two different strings:

```suggestion
        mock_middleware_random.side_effect = ['examplenonce', 'examplestring']
```

Reply:
This has multiple occurrences so I will follow up with a different PR to change all of them